server/app/views/components/ProgramCardFactory.java

```diff
@@ -165,11 +180,16 @@ private static ProgramDefinition getDisplayProgram(ProgramCardData cardData) {
 
   public static Comparator<ProgramCardData> lastModifiedTimeThenNameComparator() {
```

Comment:
Can we update the name of this comparator so it reflects the additional behavior of sorting the common intake form above everything else?